.github/workflows/ci.generate.ts

```diff
@@ -229,7 +253,7 @@ const ci = {
         CARGO_TERM_COLOR: "always",
         RUST_BACKTRACE: "full",
       },
-      steps: [
+      steps: cancelTestReleaseJobsIfPR([
```

Comment:
Oh, there's some conditions within here I can remove now. Will look into it soon

Reply:
Didn't see anything obvious.